target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -83,6 +83,10 @@ tplink,archer-c50-v6)
 	ucidef_set_led_wlan "wlan2g" "wlan2g" "green:wlan2g" "phy0tpt"
 	ucidef_set_led_wlan "wlan5g" "wlan5g" "green:wlan5g" "phy1tpt"
 	;;
+tplink,archer-mr200-v5)
+	ucidef_set_led_netdev "lan" "lan" "white:lan" "eth0.1"
```

Comment:
Why do you trigger the LED on `eth0.1` and not `eth0`?
The `.1` will link it to a vlan interface.